lib/beaker/dsl.rb

```diff
@@ -1,5 +1,5 @@
-[ 'install_utils', 'roles', 'outcomes', 'assertions', 'patterns',
-  'structure', 'helpers', 'wrappers', 'test_tagging' ].each do |lib|
+['install_utils', 'roles', 'outcomes', 'assertions', 'patterns',
+ 'structure', 'helpers', 'wrappers', 'test_tagging',].each do |lib|
```

Comment:
This trailing comma also looks odd.

Reply:
I was able to fix this by moving it to a single line. it's 132 chars long and 140 is the current limit. As an alternative, we could disable the cop for that two lines.

Reply:
In these cases I generally put the `]` on it's own line (not sure about the rules we have for spacing, but it is not my point here):

```ruby
foo = [
        1,
        2,
        3,
      ]
```